website/content/v1.10/talos-guides/install/single-board-computers/rpi_generic.md

```diff
@@ -61,11 +61,53 @@ Now `dd` the image to your SD card:
 sudo dd if=metal-arm64.raw of=/dev/mmcblk0 conv=fsync bs=4M
 ```
 
+## [OPTIONAL] Updating 'config.txt' for V3D/VC4 Broadcom VideoCore GPU support
+
+The default `config.txt` from Talos must be edited to contain specific options to enable Broadcom VideoCore GPU support.
```

Comment:
can avoid and say just generate a new image from Image Factory with the required customization, i.e the `vc4` extension and the `configTxtAppend` an d use that image to install/upgrade

Reply:
since the mounting and editing of `config.txt` won't survive an upgrade